Translated/Story/Side Aina/ssc_501110.csv

```diff
@@ -0,0 +1,34 @@
+ssc_501110_0010#0,"""Phew... It's so hot out here!"""
+ssc_501110_0020#0,"""It's not dry like Retem's air...<br>It's like there's heat from head-to-toe."""
+ssc_501110_0030#0,"""<%me>, does the heat really not bother you here?"""
+ssc_501110_0040#0,"""Nope."""
+ssc_501110_0050#0,"""Honestly, it kind of does."""
+ssc_501110_0060#0,"""I see..."""
+ssc_501110_0070#0,"""I guess it really must be<br>a Meteorn thing or something..."""
+ssc_501110_0080#0,"""What? It does?"""
+ssc_501110_0090#0,"""You seemed totally fine<br>in Retem and Kvaris, though."""
+ssc_501110_0100#0,"""Well, I guess it's a good thing, since<br>we can go to a bunch of different places..."""
```

Comment:
Could you give this line a second look? It's mostly fine following from line 7, but it doesn't make sense coming from line 9. And the 'can' in the second half of is unclear whether it means "we could be sent to" or "your hardiness means we can go to"

Reply:
it could be either, so i just picked one. fixt